integration-tests/bulkExport.test.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import axios from 'axios';
+import { groupBy, mapValues, uniq, isEqual } from 'lodash';
+import createBundle from './createPatientPractitionerEncounterBundle.json';
+import BulkExportTestHelper, { ExportStatusOutput } from './bulkExportTestHelper';
+
+const FIVE_MINUTES_IN_MS = 5 * 60 * 1000;
+const { API_URL, API_ACCESS_TOKEN, API_KEY } = process.env;
+
+describe('Bulk Export', () => {
+    let bulkExportTestHelper: BulkExportTestHelper;
+
+    beforeAll(() => {
+        if (API_URL === undefined) {
+            throw new Error('API_URL environment variable is not defined');
+        }
+        if (API_ACCESS_TOKEN === undefined) {
+            throw new Error('API_ACCESS_TOKEN environment variable is not defined');
+        }
```

Comment:
The `accessToken` is provided as an environment variable. For CI/CD with github actions we generate it on the command line and pass it into the test. However, for running this locally, I think it would be more convenient if we don't have to manually generate the token each time. 

I'll add Cognito so that the access token can be auto generated.